examples/NeoPixelBrightness/NeoPixelBrightness.ino

```diff
@@ -75,6 +75,10 @@ void loop()
     }
     // apply dimming
     brightness += direction;
+
```

Comment:
add a comment about reseting the colors due to the descructive nature of brightness bus